model/src/ww3_ounp.F90

```diff
@@ -2158,7 +2158,7 @@ SUBROUTINE W3EXNC(I,NCID,NREQ,INDREQ,ORDER)
                  RHOAIR, USTAR, USTD, Z0, CD, CHARN )
 #endif
             !
-            DO ITT=1, 3
+            DO ITT=1, 4
```

Comment:
@mickaelaccensi I was looking through these updates and saw this one line change that seemed a bit out of place given everything else.  My guess is  this likely isn't the issue causing the unexpected diffs, but wanted to point it out just in case.